src/models/Affiliation.js

```diff
@@ -0,0 +1,37 @@
+module.exports = function (sequelize, DataTypes) {
+  const Affiliation = sequelize.define(
+    'AffiliationModel',
+    {
+      cv_id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        references: {
+          model: 'CongresspersonModel',
+          key: 'cv_id',
+        },
+      },
+      active: DataTypes.BOOLEAN,
+      political_party: DataTypes.TEXT,
+      political_party_scope: DataTypes.TEXT,
+      political_party_status: DataTypes.TEXT,
+      political_party_registration_date: DataTypes.TEXT,
+      affiliation_status: DataTypes.TEXT,
+      affiliation_begin: DataTypes.TEXT,
+      affiliation_cancellation: DataTypes.TEXT,
+      affiliation_representative: DataTypes.TEXT,
+      affiliation_committee: DataTypes.TEXT,
+    },
+    {
+      tableName: 'affiliation',
```

Comment:
Agregar `timestamps: false` para evitar que busque el valor de `createdAt` y `updatedAt` si es que no existen en la tabla

Reply:
dale ahora lo corrijo

Reply:
solved in 1417cfb